src/components/avatar/index.ts

```diff
@@ -27,10 +28,13 @@ export default class Avatar extends Block<AvatarProps>{
                 if (props.path || props.url)
                 {
                     //TODO: change meta url
+                    const defaultPath = `https://ya-praktikum.tech/api/v2/resources/`
                     let imageUrl;
                     if (props.path)
-                        imageUrl = import(props.path);
-                    this.element.style.backgroundImage = `url(${props.url? props.url :  imageUrl})`;
+                        imageUrl = defaultPath + props.path;
+                    // console.log('!', imageUrl)
+                    this.element.style.backgroundImage = `url(${imageUrl ? imageUrl :  props.url})`;
```

Comment:
Можно использовать сокращенную форму
```suggestion
                    this.element.style.backgroundImage = `url(${imageUrl ??  props.url})`;
```